action.yml

```diff
@@ -1,5 +1,5 @@
-name: 'ftp-action'
-author: 'Sebastian Popp <ftp-action@sebastianpopp.com>'
+name: 'ftp-action-delete-first'
+author: 'hsimah'
```

Comment:
... and revert `name` and `author`

Reply:
🤦 I did not mean to include these in the PR; let me revert that commit